core/math/geometry_2d.cpp

```diff
@@ -0,0 +1,384 @@
+/*************************************************************************/
+/*  geometry.cpp                                                         */
```

Comment:
This should be `geometry_2d.cpp` (beware of the trailing `*/` then, three spaces should be removed).

Reply:
But actually no need to rebase and stress out CI, there are other header issues in the current `master` branch so I'll fix them all at once.

Reply:
Ah yes it's easy to miss something like this sorry... Yeah I won't rebase then.